tools/Performance/DynamoPerformanceTests/PerformanceTestHelper.cs

```diff
@@ -142,5 +144,18 @@ public FastBenchmarkReleaseConfig() : base()
                 );
             }
         }
+
+        /// <summary>
+        /// Gets the absolute path for a relative path
```

Comment:
maybe explain this is relative to the executing assembly, otherwise it seems magical :)